grants/milestone_deliveries/season_2/package_manager/milestone1-part1.md

```diff
@@ -0,0 +1,39 @@
+# Milestone Delivery :mailbox: - Milestone 1, Part 1
+
+**The [invoice form :pencil:](TODO) has been filled out correctly for this milestone and the delivery is according to the official [milestone delivery guidelines](https://github.com/radicle-dev/radicle-grants/blob/main/docs/deliverables_guideline.md).**  
+
+* **Application Document:** [Grant Application - Package Manager](https://radicle.community/t/grant-application-package-manager/3122)
+* **Milestone Number:** Milestone 1, part 1
+
+**Context**
+
+The scope of this first part of the first milestone was to *Generate an npm-compatible package from source code obtained from Radicle*. While issues were discovered during this research, a solution for moving forward was found.
+
+**Deliverables**
+
+The [research notes](./milestone1-part1-details.md) are presented here as a deliverable for the first part of the first milestone. A summary video is included, in addition to written notes. It was determined that it **is** possible to build a JavaScript application without any dependence on GitHub or npm. JavaScript packages (dependencies) can be retrieved directly from Radicle Seed nodes (as git repositories).
+
+| Number | Deliverable | Link | Notes |
+| ------------- | ------------- | ------------- |------------- |
+| 1. | Generate an npm-compatible package from source code obtained from Radicle | [Details](./milestone1-part1-details.md) | Research notes |
+
+**Additional Information**
+
+Additional guidance is needed before moving on to the second and third parts of the fist milestone, but it is possible to move on directly to the fourth part. Here are the four parts of the first milestone:
```

Comment:
What guidance is needed?

I think it'd be good to get input from Clients team here

Reply:
Right, I should have been clearer. The guidance needed is: What does 'signing' and 'uploading' mean within the context of git repositories? The assumption was working with binary files, but if that is not something to explore at this time, then those two steps may not be needed.

Since Seed nodes already have a signing mechanism, I don't think we need to explore additional signing. And uploading of files to a git repository is not something that needs to be explored.

If the feature of fetching individual files is developed, then these two areas of research could be explored.